src/posts/pangeo-ml-ecosystem-2023/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: 'The Pangeo Machine Learning Ecosystem in 2023'
+date: '2023-11-02'
+authors:
+  - name: Wei Ji Leong
+    github: weiji14
+  - name: Max Jones
+    github: maxrjones
+  - name: Negin Sobhani
+    github: negin513
+  - name: Deepak Cherian
+    github: dcherian
```

Comment:
@maxrjones, @negin513, @dcherian, I've put you down as co-authors for all your work on xbatcher/cupy-xarray/kvikIO. Let me know if you're happy with that!